src/shared/varlink.c

```diff
@@ -281,22 +281,22 @@ int varlink_connect_address(Varlink **ret, const char *address) {
 
         r = sockaddr_un_set_path(&sockaddr.un, address);
         if (r < 0)
-                return r;
+                return log_debug_errno(r, "Failed to set socket address '%s': %m", address);
         sockaddr_len = r;
 
         r = varlink_new(&v);
         if (r < 0)
-                return r;
+                return log_debug_errno(r, "Failed to create varlink object: %m");
```

Comment:
`log_oom_debug()`?

Reply:
I considered that, but i thought that varlink may also fail for other reasons in the future, so it's better to not hardcode the failure reason here.